EMacco/abarian-backend
# #170468585 Fetch products by category

#### What does this PR do?
This PR allows a user to be able to fetch a list of products under a category

#### Description of Task to be completed?
- return all products that are under a particular category
- make sure category exist

#### How should this be manually tested?
- checkout to this branch 
- send a get request to `/products/categories/:categoryId?page=page_number&limit=limit_number`
- an array of products should be returned

#### Any background context you want to provide?
- category should exist

#### What are the relevant pivotal tracker stories?
[#170468585](https://www.pivotaltracker.com/story/show/170468585)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
